virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc

```diff
@@ -11,12 +11,8 @@ You can create virtual machines (VMs) from instance types by using the {product-
 ifdef::openshift-rosa,openshift-dedicated[]
 [NOTE]
 ====
-Creating a VM from an instance type in {VirtProductName} 4.15 and higher is supported for use on {product-title} clusters. In {VirtProductName} 4.14, Creating a VM from an instance type is a Technology Preview feature, and is not supported for use on {product-title} clusters
+Creating a VM from an instance type in {VirtProductName} 4.15 and higher is supported for use on {product-title} clusters. In {VirtProductName} 4.14, Creating a VM from an instance type is a Technology Preview feature, and is not supported for use on {product-title} clusters.
```

Comment:
We need to use a [tech preview snippet](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features) as per guidelines.

```suggestion
Creating a VM from an instance type in {VirtProductName} 4.15 and higher is supported for use on {product-title} clusters. In {VirtProductName} 4.14, creating a VM from an instance type is a Technology Preview feature, and is not supported for use on {product-title} clusters.
```

Reply:
Good catch on that capitalization, thanks! I don't think the TP snippet is necessary here though. My understanding is that because the ROSA docs are versionless, this note is more to say that you should check your OCP Virt version before using instance types. If your CNV version is 4.15, you're good (it's fully supported) but if you are using 4.14 _with ROSA_, instance types are *not* supported because tech preview features are not supported in ROSA. Is that accurate, @mburke5678 ?

Reply:
@ousleyp @snarayan-redhat Correct. Tech preview features are not supported in ROSA (or OSD).